.gitignore

```diff
@@ -117,4 +117,7 @@ dmypy.json
 dump.rdb
 
 # Redis
-*.rdb
\ No newline at end of file
+*.rdb
```

Comment:
after you will be added to the wiremind organisation, create pull requests directly in the repository. It's easier for me to checkout your work.